frontend/src/components/search.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+class Search extends React.Component {
+
+  componentDidUpdate(prevProps, prevState) {
+
+  }
```

Comment:
This componentDidUpdate function does nothing and should have been omitted -- whoops